pkg/flavor/flavor_test.go

```diff
@@ -76,18 +95,35 @@ var _ = Describe("Flavor and Preferences", func() {
 			Expect(spec).To(BeNil())
 		})
 
+		It("store returns error when no flavorMatcher kind is invalid", func() {
```

Comment:
The description contains a typo, it should probably be: `store returns error when flavorMatcher kind is invalid`

Reply:
Done.